VideoLocker/src/main/java/org/edx/mobile/module/notification/EdxParsePushBroadcastReceiver.java

```diff
@@ -32,101 +33,129 @@
     //UI and animation time need to get from production team.
     private boolean enableFlyinWindow = false;
```

Comment:
I don't even know why this is in the code-base. I'd like to just delete this "flyin" concept, whatever it is. It's disabled, and the service it references is commented out. @aleffert ?